asm_to_pil/src/vm_to_constrained.rs

```diff
@@ -1029,11 +1027,17 @@ fn extract_update<T: FieldElement>(expr: Expression<T>) -> (Option<String>, Expr
     // TODO check that there are no other "next" references in the expression
```

Comment:
Does this comment need to be updated?

Reply:
No, we are still not checking the inner of the expression.